authpass/lib/ui/screens/password_list.dart

```diff
@@ -923,11 +939,49 @@ class _PasswordListContentState extends State<PasswordListContent>
     return null;
   }
 
+  List<Widget>? _buildAutofillSuggestBanners() {
+    if (_autofillStatus == null ||
+        _autofillStatus != AutofillServiceStatus.disabled) {
+      return null;
+    }
+    if (_dismissedAutofillSuggestion != null &&
+        _dismissedAutofillSuggestion == true) {
+      return null;
+    }
+    final loc = AppLocalizations.of(context);
+    return [
+      MaterialBanner(
+        content: Text(loc.enableAutofillSuggestionBanner),
+        actions: [
+          TextButton(
+            onPressed: () {
+              context.read<AppDataBloc>().update((builder, data) {
+                builder.dismissedAutofillSuggestion = true;
+              });
+            },
+            child: Text(loc.dismissAutofillSuggestionBannerButton),
+          ),
+          TextButton(
+            onPressed: () {
+              _autofill.requestSetAutofillService().then((result) {
+                _autofill.status().then((value) {
+                  _autofillStatus = value;
+                });
+              });
+            },
```

Comment:
i think i'd prefer using async/await syntax, is a bit more consice:
```suggestion
            onPressed: () async {
              await _autofill.requestSetAutofillService();
              await _updateAutofillPrefs();
            },
```